### DISCUSSION TITLE
How to remove the new page down and page up animation?

### DISCUSSION THREAD

**Discussion by rafa-thayto** [LazyVim/LazyVim#5045] [General]
## Description
How to remove the new page down and page up animation?

Would anyone be able to help me?

**Comment by folke** [comment]
You start by asking nicely without `?????`

I added it.

Disable with `vim.g.snacks_animate = false`

You're welcome

**Reply by rafa-thayto** [reply]
Sorry folke, but this animation is stressing me out a lot. I apologize. :heart:

**Reply by folke** [reply]
Users talking to me like this are stressing me out a lot. Sorry
